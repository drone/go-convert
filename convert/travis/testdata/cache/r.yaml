# https://docs.travis-ci.com/user/caching/#r-package-cache

language: R

cache: packages