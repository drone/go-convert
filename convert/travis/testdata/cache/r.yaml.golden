kind: pipeline
spec:
  stages:
  - desc: converted from travis.yml
    name: pipeline
    spec:
      cache:
        enabled: true
        paths:
        - ~/R/Library
    type: ci
version: 1
